Source/Layout/BrickLayoutSection.swift

```diff
@@ -85,6 +85,10 @@ internal class BrickLayoutSection {
     /// Width of the section. Can be set by `setSectionWidth`
     /// This is not a calculated property of the frame because in case of horizontal scrolling, the values will be different
     internal private(set) var sectionWidth: CGFloat
+    
+    /// Height of the section.
+    /// This is not a calculated property of the frame because in case of vertical scrolling, the values will be different
+    internal private(set) var sectionHeight: CGFloat?
```

Comment:
Isn't the first attribute in the `attributes` variable, the place to get that from?